### ISSUE TITLE
Enable PipeWire capturer by default

### ISSUE THREAD

**Issue by vchernin** [flathub/im.riot.Riot#202]
Now that Element 1.8 is released (should be merged here in #201), screen sharing now works in 1:1 calls. Group calls (embedded Jitsi) are still broken due to #150.

However, for Wayland we need to pass `--enable-features=WebRTCPipeWireCapturer` at runtime. This could be done in the desktop file or maybe with zypak. As a workaround you can do `flatpak run im.riot.Riot --enable-features=WebRTCPipeWireCapturer`.

I tried to build locally but apparently flatpak-builder on Fedora 34 Silverblue for me is cursed with OOM errors (this has happened elsewhere). I'll open a PR after #201 is merged.

We can also remove PipeWire 0.2.7 from the manifest since [Element now ships Electron 13](https://github.com/vector-im/element-web/issues/17833) (which works with 0.3.x in the runtime).

I also noticed Element [experiences a similar screen sharing with PipeWire issue as Jitsi meet electron](https://github.com/jitsi/jitsi-meet-electron/issues/608) (nothing to do with Element, but another electron 13 app).

**Comment by vchernin** [comment]
Opened an issue to enable the flag at buildtime as that will help .DEB users out too: https://github.com/vector-im/element-web/issues/18607

**Comment by vchernin** [comment]
Open PR: https://github.com/vector-im/element-desktop/pull/256

**Comment by vchernin** [comment]
This is now fixed upstream, and should work out of the box in the next Element release. See https://github.com/vector-im/element-web/issues/18607#issuecomment-902304958

I'll open a followup PR here to remove the old PipeWire and remove the metainfo mention of PipeWire (it is redudant now).

**Comment by vchernin** [comment]
Follow up issue: https://github.com/vector-im/element-web/issues/18647
